src/main/java/com/exasol/dbbuilder/dialects/postgres/PostgreSqlImmediateDatabaseObjectWriter.java

```diff
@@ -0,0 +1,41 @@
+package com.exasol.dbbuilder.dialects.postgres;
+
+import java.sql.Connection;
+
+import com.exasol.dbbuilder.dialects.*;
+
+/**
+ * PostgreSQL {@link DatabaseObjectWriter}.
+ */
+public class PostgreSqlImmediateDatabaseObjectWriter extends AbstractImmediateDatabaseObjectWriter {
+
+    /**
+     * Create a new instance of {@link PostgreSqlImmediateDatabaseObjectWriter}.
+     * 
+     * @param connectionToPostgres connection to the PostgreSQL database.
+     */
+    public PostgreSqlImmediateDatabaseObjectWriter(final Connection connectionToPostgres) {
+        super(connectionToPostgres);
+    }
+
+    @Override
+    protected String getQuotedColumnName(final String columnName) {
+        return PostgreSqlIdentifier.of(columnName).quote();
+    }
+
+    @Override
+    public void write(final User user) {
+        writeToObject(user, "CREATE USER " + user.getFullyQualifiedName() + " PASSWORD '" + user.getPassword() + "'");
+    }
+
+    @Override
+    public void write(final User user, final GlobalPrivilege... privileges) {
+        throw new UnsupportedOperationException(
```

Comment:
Extending a class restricting the super class is usually not a good idea, as it breaks the Liskov substitution principle.
A class should should extend another one if:
- subclass is a superclass
- subclass can do everything the superclass can

If not, there's a problem with the hierarchy and should be changed. I would recommend to address this issue.

Reply:
I'll create a ticket